splink/spark/enable_splink.py

```diff
@@ -0,0 +1,31 @@
+from splink.spark.jar_location import similarity_jar_location
+
+def enable_splink(spark):
```

Comment:
Is it possible to have a very detailed docstring on this function if possible please?

Reply:
In particular what it would be great to also mention whether this is a Databricks platform only function. In this case perhaps it should be perhaps located under `root/splink/databricks/`

Reply:
Agreed - we need a mechanism of making it clear that this function is for databricks users only.  Happy with either of the following two options (or anything else you think makes sense):
- put this file in a new folder, so it's at `splink/spark/databricks/enable_splink.py`
- and/or rename the function to `enable_splink_in_databricks`

Yes - and would also be useful to have a docstring that explains what this does.  Thanks very much!

Reply:
good idea. I think I prefer the adding a new folder approach - it doesn't need to live with the SparkLinker. 

I'll refactor and document later on today